environment.py

```diff
@@ -362,11 +359,18 @@ def env() -> Mapping[str, Optional[str]]:
         # Typically only set for main deployments.
         'AZUL_ENABLE_MONITORING': '0',
 
-        # The default bundle UUID prefix to use for reindexing bundles in the DSS
-        # and for subscriptions to the DSS. If this variable is set to a non-empty
-        # string, only bundles whose UUID starts with the specified string will be
-        # indexed.
-        'AZUL_DSS_QUERY_PREFIX': '',
+        # Identifies the DSS Repository endpoint and prefix to index.
+        # The syntax in EBNF is:
+        #
+        # source = 'https://', URL,
+        #          '/', BASE PATH,
+        #          ':', [ UUID prefix [ '/', Partition prefix length ] ] ;
```

Comment:
If you want to be formal, then be formal. This is half-baked. `URL`, `BASE PATH`, `UUID prefix` and `Partition prefix length` are all undefined non-terminals. Why do you all-caps some but not others? 

What you call "URL" is a host name or, more precisely, in terms of the RFC, a netloc. 

https://stackoverflow.com/questions/53992694/what-does-netloc-mean

Then again, attempting to restrict/disseminate the syntax of a URL overkill for this.

Reply:
note the comment on SO modified the formatting for `net_loc` to `netloc` opted to use the formatting as described in the RFC.
https://www.rfc-editor.org/rfc/rfc1808.html#section-2.1